src/nidigital/system_tests/test_system_nidigital.py

```diff
@@ -638,6 +639,23 @@ def test_configure_active_load_levels(multi_instrument_session):
     assert multi_instrument_session.pins['site0/PinA', 'site1/PinC'].active_load_vcom == pytest.approx(3.0, rel=1e-3)
 
 
+def test_disable_sites(multi_instrument_session):
+    # Also tests is_site_enabled
+    multi_instrument_session.load_pin_map(os.path.join(test_files_base_dir, "pin_map.pinmap"))
+    assert multi_instrument_session.sites[1].is_site_enabled()
+    multi_instrument_session.sites[1].disable_sites()
```

Comment:
Not technically required, but I think it would be good to test calling `disable_sites` and `enable_sites` with multiple sites.

Reply:
Done.
I reorganized the tests and changed the code a little bit to test every case that you might be interested in.